pkg/util/informermanager/single-cluster-manager.go

```diff
@@ -54,6 +58,12 @@ type SingleClusterInformerManager interface {
 
 	// GetClient returns the dynamic client.
 	GetClient() dynamic.Interface
+
+	ListerWithNode() v1.NodeLister
```

Comment:
Should we put the lister of node and pod to `single-cluster-manager.go` ,It is a util for dynamic informer. This seems a little strange.

Reply:
Yes, I agree, but In a large-scale scenes (e.g. 50w Nodes & 200w Pods), The Dynamic informer Sync the Resource (especially Nodes and Pods) will consume a lot of memory and CPU, I have tried my best not to modify the original interface and method(because many controllers use the `single-cluster-manager`), but it does look strange now. Could you please give me some advice, thx.

Reply:
Just a questiton and I have no objection to it, may be we can ask @Poor12 for suggestion

Reply:
If we only need lister of node and pod in `cluster_status_controller`,we can put listers in it